vm-ubuntu-20.04/user-common-bootstrap.sh

```diff
@@ -17,11 +17,6 @@ sudo ./mininet/util/install.sh -nw
 
 find /usr/lib /usr/local $HOME/.local | sort > $HOME/usr-local-7-after-mininet-install.txt
 
-# --- Tutorials --- #
-git clone https://github.com/p4lang/tutorials
-sudo mv tutorials /home/p4
-sudo chown -R p4:p4 /home/p4/tutorials
-
```

Comment:
@sc68cal There is a list with pre-built VM images that is regularly updated using this bootstrap script:
https://github.com/jafingerhut/p4-guide/blob/master/bin/README-install-troubleshooting.md
Removing the tutorials code from these images would indeed be inconvenient for users VM images.

> I prefer to use the editor and development environment on the machine that is running vagrant and then just ssh into the vagrant machine and execute the commands.

If you use Linux, you can install all dependencies and tools in your development environment, or perhaps use something like [p4app](https://github.com/p4lang/p4app) or [p4-utils](https://github.com/nsg-ethz/p4-utils).